AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -47,6 +47,10 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignItems) {
   ASStackLayoutAlignItemsCenter,
   /** Expand children to fill cross axis */
   ASStackLayoutAlignItemsStretch,
+  /** Children align to their first baseline. Only available for horizontal stack nodes */
+  ASStackLayoutAlignItemsBaselineFirst,
```

Comment:
To be clear, the first / last thing could only affect text nodes, right?  ASLayoutable has the baseline defined in a direct / literal sense, without a concept of multiple lines or wrapping.  Would it make sense to simplify this to AlignItemsBaseline, and then have logic / properties on ASTextNode that controls whether it a) sets nothing, b) sets to the first baseline, c) sets to the last baseline?